itests/org.openhab.binding.wemo.tests/src/main/java/org/openhab/binding/wemo/internal/handler/test/WemoHandlerOSGiTest.java

```diff
@@ -107,12 +108,11 @@ public void assertThatThingHandlesOnOffCommandCorrectly()
     public void assertThatThingHandlesREFRESHCommandCorrectly()
             throws MalformedURLException, URISyntaxException, ValidationException {
         Command command = RefreshType.REFRESH;
+        String host = "127.0.0.1";
 
         Thing thing = createThing(THING_TYPE_UID, DEFAULT_TEST_CHANNEL, DEFAULT_TEST_CHANNEL_TYPE);
 
-        waitForAssert(() -> {
-            assertThat(thing.getStatus(), is(ThingStatus.ONLINE));
-        });
+        assertThat(thing.getStatus(), is(ThingStatus.ONLINE));
```

Comment:
Did something change so that test no longer needs to wait for thing going online?